general/releases.rst

```diff
@@ -0,0 +1,146 @@
+.. _releases:
+
+Releases
+########
+
+.. _releases_cadance:
+
+Cadence
+*******
+New versions of OP-TEE are released four times a year, i.e., quarterly releases.
+The releases have historically lined up with Linaro Connect events. I.e., a
+release has been made right after Linaro Connect and one release somewhere
+between two Linaro Connects. I.e., typically there has been releases in January,
+April, July and October.
+
+.. _releases_changelog:
+
+Changelog
+*********
+The changelog is stored in the :ref:`optee_os` git (CHANGELOG.md_). There you
+can see what has been done between the different releases in terms of commits as
+well as pull requests. It's worth noting that this changelog only lists changes
+in optee_os and not the other optee_xyz gits. However, most work is done in that
+git and often changes in optee_os also require changes in :ref:`optee_client`
+and :ref:`optee_test`.
+
+.. _releases_versioning_schema:
+
+Versioning schema
+*****************
+OP-TEE follows `Semantic Versioning 2.0.0`_. What that means in practice is well
+described at the link just shown.
+
+.. _releases_release_procedure:
+
+Release procedure
+*****************
+There are certain steps that needs to be done when making a release. This
+checklist here serves as guidance to the one in charge of making a new release.
+Roughly start with this 2-3 weeks before the targeted release date.
+
+tl;dr
+=====
+.. list-table:: Short version of the OP-TEE release procedure
+    :widths: 60 300 10
+    :header-rows: 1
+
+    * - When
+        (Tminus)
+      - Action
+      - Example
+
+    * - 3w
+      - Create release pull request
+      - `PR#3099`_
+
+    * - 3w
+      - Inform maintainers about upcoming release
+      -
+
+    * - 1w
+      - Create RC-x tag in optee_* + build.git
+      - git tag -a 3.x.y-rc1 -m "3.x.y-rc1"
```

Comment:
Intentionally left out hypens here, since that makes the entire table to quite bad. Rst doesn't wrap "code", so the cell is stretched out to whatever is needed which compresses the other cells heavily.